composeApp/src/dwMain/kotlin/org/ooni/probe/ui/Colors.kt

```diff
@@ -0,0 +1,16 @@
+package org.ooni.probe.ui
+
+import androidx.compose.ui.graphics.Color
+
+val primaryColor = Color(0xffD32625)
+val primaryLightColor = primaryColor.copy(alpha = 0.75f)
+val secondaryColor = Color(0xff5db8fe)
+val secondaryLightColor = secondaryColor.copy(alpha = 0.75f)
+val primaryTextColor = Color(0xffffffff)
+val secondaryTextColor = Color(0xff000000)
+val surfaceDark = Color(0xFF161616)
+val surfaceLight = Color(0xFFFFFFFF)
+val backgroundLightColor = Color(0xffF1F0F5)
+val backgroundDarkColor = Color(0xff010100)
+val errorColor = Color(0xFFFF8989)
+val onErrorColor = Color(0xFF000000)
```

Comment:
To make sure we're not missing any color on a specific flavor, we might need to wrap them in a class.